blockscout/.env.example.local-devnet

```diff
@@ -0,0 +1,6 @@
+# local devnet
+BASE_EXPLORER_DOMAIN=evm.kadena.local
```

Comment:
Can we avoid using `.local` tld? It's a reserved TLD for [mDNS](https://en.wikipedia.org/wiki/Multicast_DNS#:~:text=By%20default%2C%20mDNS,designed%20to%20avoid.)